src/main/kotlin/org/rust/cargo/project/workspace/CargoWorkspace.kt

```diff
@@ -83,6 +85,11 @@ interface CargoWorkspace {
         val edition: Edition
     }
 
+    interface Dependency {
+        val pkg: Package
+        val name: String?
```

Comment:
This should be non-null and set in constructor from `pkg` if an epxlicit name is missing?

Reply:
Good idea!